src/core/eme/index.ts

```diff
@@ -171,15 +171,25 @@ function dispose() : void {
 /**
  * Clear EME ressources as the current content stops its playback.
  */
-function clearEME(): Observable<MediaKeys> {
+function clearEME(): Observable<never> {
   return Observable.defer(() => {
+    const observablesArray : Array<Observable<never>> = [];
     if (instanceInfos.$videoElement && shouldUnsetMediaKeys()) {
-      return disposeMediaKeys(instanceInfos.$videoElement)
+      const obs$ = disposeMediaKeys(instanceInfos.$videoElement)
+        .ignoreElements()
         .finally(() => {
           instanceInfos.$videoElement = null;
-        });
+        }) as Observable<never>;
+      observablesArray.push(obs$);
     }
-    return Observable.empty();
+    if (instanceInfos.$keySystem && instanceInfos.$keySystem.closeSessionsOnStop) {
+      observablesArray.push(
+        $loadedSessions.dispose()
+          .ignoreElements() as Observable<never>
```

Comment:
same

Reply:
:+1:  This is actually scheduled for the next major of RxJS:
https://github.com/ReactiveX/rxjs/pull/2691

:raised_hands: